prow/tide/README.md

```diff
@@ -0,0 +1,128 @@
+# Tide
+
+One of [Prow](https://github.com/kubernetes/test-infra/blob/master/prow/README.md)
```

Comment:
s/One of/Tide is a/

We don't have any other Prow components that do this.

Reply:
Done